src/controllers/userController.ts

```diff
@@ -495,6 +503,82 @@ const update_user_profile = async (req: Request, res: Response) => {
 	}
 };
 
+export const accountStatus = async (req: Request, res: Response) => {
+	const userId = req.params.userId;
+	const { isAccountActive, reason } = req.body;
```

Comment:
@denislohan can you assist us here, I don't think is important to provide the reason when disabling the user account, I think we can remove the body completely then when admin hit endpoint we change the status from the pervious one . or we can take this PR as it is now?